atlp-rwanda/atlp-devpulse-fn
# Ft shareable job post

## PR Description

This PR implements the shareable post page feature, and fix for the single job display

## Description of tasks that were expected to be completed

- Fix shareable post page
- Redirect to link
- Add shareable links for different social media platforms
- Fix single job display

## How has this been tested?

- Clone repo
- Checkout branch `ft-shareable-job-post`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully

## How to reproduce

1. Login
2. Click on View Job Post on the sidebar
3. On the dropdown, click view

![Screenshot 2024-09-25 111852](https://github.com/user-attachments/assets/f0d5492f-6851-4a69-a17c-f5e4ecfa9919)


4. On the shareable post page, click button for sharing on whatsapp, twitter, telegram, gmail or linkedin, you will be redirected to the respective pages

![Screenshot 2024-09-25 111933](https://github.com/user-attachments/assets/46bf33d0-1484-4cfe-821a-db4873dccd1a)


## Contributors

@SHYAKA-Aime 
@bahati10
